### ISSUE TITLE
Failing Conformance Testing 20

### ISSUE THREAD

**Issue by tvarkey007** [EIPStackGroup/OpENer#500]
I noticed that OpENer is failing the conformance test on the latest conformance test software (CT20). When reading up I figured out that OpENer hasn't been updated to meet the latest spec for ODVA. Is it possible to get a estimated timeframe for OpENer to be updated in accordance with the latest ODVA spec?

**Comment by MartinMelikMerkumians** [comment]
No, there is no timeframe, however everyone is invited to provide code contributions to make that happen
